packages/storybook/stories/va-search.stories.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { getWebComponentDocs, propStructure } from './wc-helpers';
+import { generateEventsDescription } from './events';
+
+const searchDocs = getWebComponentDocs('va-search');
+
+export default {
+  title: 'Components/va-search',
+  parameters: {
+    docs: {
+      description: {
+        component: generateEventsDescription(searchDocs),
+      },
+    },
+  },
+  // argTypes: {
+  //   method: {
+  //     control: {
+  //       type: 'select',
+  //       options: ['GET', 'POST', 'dialog'],
+  //     },
+  //   },
+  // },
+};
+
+const Template = ({ action, label }) => (
+  <va-search
+    action={action}
+    label={label}
+    // method={method}
+    onBlur={() => console.log('BLUR')}
+    onChange={() => console.log('CHANGE')}
+    onClick={() => console.log('CLICK')}
+    onKeyDown={() => console.log('KEYDOWN')}
+  />
+);
```

Comment:
And in storybook we can add a handler for the default event like this:

```jsx
const Template = ({ action, label, method, submit }) => (
  <va-search
    action={action}
    label={label}
    method={method}
    onSubmit={e => {
      console.log('Outer submit handler');
      console.log(e);
    }}
  />
);
```

Reply:
This isn't working for `onChange`.